modules/meshrenderinggl/src/processors/linerasterizer.cpp

```diff
@@ -195,13 +205,12 @@ void LineRasterization::rasterize(const ivec2& imageSize,
             for (size_t i = 0; i < mesh->getNumberOfIndicies(); ++i) {
                 if (mesh->getIndexMeshInfo(i).dt != DrawType::Lines) continue;
                 auto& shader = lineShaders_->getShader(*mesh, mesh->getIndexMeshInfo(i));
-                if (!shader.isReady()) {
-                    LogWarn("Shader not ready.");
-                    break;
-                }
+                if (!shader.isReady()) break;
 
                 shader.activate();
-                utilgl::setShaderUniforms(shader, *mesh, "geometry");
+                auto transform = CompositeTransform(mesh->getModelMatrix(),
```

Comment:
just use a SpatialEntity directly?

Reply:
As said above, that one is purely virtual.